static/base_style.css

```diff
@@ -0,0 +1,801 @@
+  font-size: 2.5rem;
+}
+
+.main_header_underline {
+  border-bottom: 2px solid #ec5242;
+  width: 8vw;
+}
+
+.main_list {
+  margin-top: 1.5rem;
+}
+
+.main_list_card {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  background: #ffffff0f;
+  margin-bottom: 1rem;
+}
+
+.main_list_img {
+  height: 45px;
+  width: 45px;
+}
+
+.main_list_header {
+  color: #ec5242;
+  font-size: 0.8rem;
+  width: 130px;
+  padding-inline: 5px;
+}
+
+.main_list_bio {
+  font-size: 0.7rem;
+}
+
+.m_link {
+  background-color: #ec5242;
+  color: #d3d3d3;
+  padding: 1.4rem 1.5rem;
+  font-size: 1.3rem;
+  margin: 1rem 0;
+}
+
+.main_link {
+  display: none;
+}
+
+.hide {
+  display: none;
+}
+
+/* about_logo section about_page */
+
+.about_logo,
+.past_event {
+  display: flex;
+  flex-direction: column;
+  color: #272a31;
+  background-color: #fff;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+.about_logo_header,
+.past_event_header {
+  color: #272a31;
+  width: 80vw;
+  padding: 1.5rem 0.5rem 0.3rem;
+}
+
+.about_logo_header_underline {
+  border-bottom: 2px solid #ec5242;
+  width: 15vw;
+}
+
+.about_logo_bio,
+.past_event_bio {
+  color: #272a31;
+  width: 90vw;
+  margin-top: 1rem;
+}
+
+.about_logo img {
+  width: 250px;
+  padding: 1rem;
+}
+
+.past_event img {
+  width: 80%;
+  margin: 1.5rem 0;
+}
+
+/* partner section about_page */
+
+.partner_container2 {
+  width: 100%;
+  background-color: #272a31;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.5rem;
+  flex-direction: column;
+}
+
+.partner_header2 {
+  padding: 1.5rem 1rem 0;
+}
+
+.partner_header_underline2 {
+  border-top: 2px solid #ec5242;
+  width: 10vw;
+  padding: 1rem 0;
+}
+
+.partner_list2 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.partner_list_item2 img {
+  width: 80px;
+}
+
+/* footer section about_page */
+
+.footer2 {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: #fff;
+  padding: 1.5rem;
+}
+
+.footer_img2 {
+  height: 50px;
+}
+
+.right_div {
+  width: 50vw;
+  line-height: 80%;
+}
+
+.rights2 {
+  color: #272a31;
+  font-size: 0.7rem;
+}
+
+/* feature speaker section home_page */
+
+.team-page {
+  flex-direction: column;
+  align-items: center;
+  padding: 0 1.2rem;
+}
+
+.subtitle {
+  color: #272a31;
+  margin: 1rem 0 0.4rem;
+}
+
+.team-card {
+  grid-template-columns: 1fr 1fr;
+  color: #272a31;
+  margin-bottom: 1.5rem;
+}
+
+.chestbg {
+  position: absolute;
+  width: 5rem;
+  height: 5rem;
+  z-index: 1;
+}
+
+.team-photo {
+  position: relative;
+  width: 8rem;
+  height: 8rem;
+  z-index: 2;
+  margin: 1.1rem 0 0 1.1rem;
+}
+
+.team-info {
+  flex-direction: column;
+  gap: 0.4rem;
+}
+
+.job-underline {
+  padding: 1px;
+  background: #272a31;
+  width: 1.8rem;
+}
+
+.job-position {
+  color: #ae2111b9;
```

Comment:
- [ ] kindly use this  `#EC5242 `color here instead as used in the original design